index.js

```diff
@@ -55,9 +54,68 @@ addBtn.addEventListener('click', (e) => {
   if (inputTitle.value.length > 0 && inputAuthor.value.length > 0) {
     bookArr.saveData(book1);
     bookArr.getData();
+  } else {
+    alert('please fill all the inputs');
   }
 });
 
 const remove = (index) => bookArr.removeBook(index);
 remove();
-bookArr.getData();
\ No newline at end of file
+bookArr.getData();
+
+// Add navigation
+
+const navListBtn = document.querySelector('#add-list');
+const addNewBtn = document.querySelector('#add-new');
+const contactBtn = document.querySelector('#contact');
+const myBookList = document.querySelector('body > main > div.awesome');
+const formInput = document.querySelector('body > main > section');
+const contactPage = document.querySelector('body > main > div.container.d-flex');
+const myTime = document.querySelector('body > main > div.set-time > p');
+const { DateTime } = this.luxon;
+
+const myListPage = () => {
+  contactPage.style.display = 'none';
+  formInput.style.display = 'none';
+  myBookList.style.display = 'block';
+};
+navListBtn.addEventListener('click', myListPage);
```

Comment:
- [ ] The Booklist page also does not follow the design. Please make sure to keep the design same as this is the hard requirement of this milestone.

### Your design

![Screenshot from 2021-11-25 19-52-58](https://user-images.githubusercontent.com/22434660/143459751-8a97482f-a91b-4551-994c-2167c33327bf.png)

### Original design

![Screenshot from 2021-11-25 19-53-16](https://user-images.githubusercontent.com/22434660/143459800-ba765781-3428-4add-9d6d-444fc77b13e9.png)